### ISSUE TITLE
 I can't register

### ISSUE THREAD

**Issue by Rasulwebs** [adrianhajdin/banking#10]
This Deployed Bank app is not working. I can't register.
I entered all the information clearly and correctly
![Screenshot 2024-05-08 173534](https://github.com/adrianhajdin/banking/assets/109275421/b989edd8-bc4c-49c4-8f6e-dfaaa5c7c979)

**Comment by othegagan** [comment]
Hey 👋,
Even i was facing the some issue with signup, hence I clone the repo and debugged it. What I found is, if we try to enter some invalid details for Postal Code, SSN, or date of birth it would result into error. Error handle as not been done yet. 
Try entering some clean data, you should be able to sign in.

**Comment by Rasulwebs** [comment]
> Hey 👋, Even i was facing the some issue with signup, hence I clone the repo and debugged it. What I found is, if we try to enter some invalid details for Postal Code, SSN, or date of birth it would result into error. Error handle as not been done yet. Try entering some clean data, you should be able to sign in.

That's it, I entered the correct job information. Gmail zip code and other details are also valid but I still can't register.

**Comment by othegagan** [comment]
I will try to include some error message, why sign up is getting failed.

**Comment by Rasulwebs** [comment]
> I will try to include some error message, why sign up is getting failed.

![Screenshot 2024-05-11 212802](https://github.com/adrianhajdin/banking/assets/109275421/d4282674-faf4-42f3-84e9-2a2ef159c68d)
![Screenshot 2024-05-11 212751](https://github.com/adrianhajdin/banking/assets/109275421/1650737e-35d9-4db4-b980-870f78f8ec1f)

The request is executed successfully, there is no error message, but the response is unclear

**Comment by othegagan** [comment]
That's because of server actions. The Error logs and messages would be the server side and will not come to client browser.

**Comment by M-ustaf-a** [comment]
It's not a signup problem; it's a  sandbox problem because sandbox does not properly create customer
![Screenshot (266)](https://github.com/adrianhajdin/banking/assets/145974580/11060c8b-d9a1-4e66-9914-48860cbcc434)

**Comment by Luttudo** [comment]
yeah that's great, now I can't  finish my school project because of this dwolla integration
